rails-commenting.rb

```diff
@@ -55,14 +62,17 @@ def destroy
       redirect_to blog_posts_path
     else
       # ---8)
+      # the else in the destory if statement saying that something is not right (the post doesnt exist or didnt specify and if directing you to specify the post id)
       redirect_to blog_post_path(@post)
     end
   end
 
   # ---9)
+  #this privte line is a hidden param that keeps these vaiables unchangeable 
```

Comment:
Unchangeable if you are not in that class. If you are in the class then it is the space where you specify which params are allowed to be changed.